### DISCUSSION TITLE
Unable to give keyboard/mouse or controller permissions since new update

### DISCUSSION THREAD

**Discussion by NaturalPotato** [m4dEngi/RemotePlayWhatever#153] [Q&A]
Ever since the newest version (0.2.7), I am unable to access the Remote Play screen. This is the screen that shows everyone in the lobby and invited, and most importantly, the way to give them keyboard/mouse and controller permissions.  Has anyone else had this? Is there a way to fix it?

**Comment by 1066Bona** [comment]
The same problem here. The Remote Play Together button is now missing. Unfortunately, you cannot transfer a mouse or keyboard to other players. Attached are two pictures, one before the patch and one after the patch. I hope you find a solution and thanks for the great little program :)
![Picture 1](https://github.com/m4dEngi/RemotePlayWhatever/assets/175050092/901912ef-5d10-4dc1-9631-4d18cd284f6f)
![picture2](https://github.com/m4dEngi/RemotePlayWhatever/assets/175050092/57211338-fb79-4793-a667-77f1fb1c5361)

**Comment by Jaquavyus** [comment]
Same issue here since update! Hopefully it gets fixed. I love this little app sooo much!

**Comment by joaquingalasso** [comment]
Same issue here! I have not been able to access that window in any way. I looked for ways to force it open but couldn't do it. If I find a way, I'll tell you.

**Comment by ProTayToeGamer** [comment]
Same thing is happening to me also

**Comment by Sergey5588** [comment]
same

**Comment by 1066Bona** [comment]
now it is running again!
